src/config-utils.test.ts

```diff
@@ -199,6 +200,51 @@ test("load non-empty input", async t => {
   });
 });
 
+test("default queries are used", async t => {
+  return await util.withTmpDir(async tmpDir => {
+    process.env['RUNNER_TEMP'] = tmpDir;
+    process.env['GITHUB_WORKSPACE'] = tmpDir;
+
+    // Check that the default behaviour is to add the default queries.
+    // In this case if a config file is specified by does not include
+    // the disable-default-queries field.
+    // We determine this by whether CodeQL.resolveQueries is called
+    // with the correct arguments.
+
+    const resolveQueriesArgs: {queries: string[], extraSearchPath: string | undefined}[] = [];
+    CodeQL.setCodeQL({
+      resolveQueries: async function(queries: string[], extraSearchPath: string | undefined) {
+        resolveQueriesArgs.push({queries, extraSearchPath});
+        return {
+          byLanguage: {
+            'javascript': {},
+          },
+          noDeclaredLanguage: {},
+          multipleDeclaredLanguages: {},
+        };
+      },
+    });
+
+    // Just create a generic config object with non-default values for all fields
```

Comment:
```suggestion
    // Just create a generic config object with a non-default value for one of the fields
```

In the [other place](https://github.com/github/codeql-action/blob/main/src/config-utils.test.ts#L170-L180) where you use this comment, you specify way more fields, hence my suggested change.

Reply:
Thanks. That comment was copy-pasted and I didn't spot it was now incorrect. I've updated to say the actual important points of this config.